packages/admin/src/routes/entities/components/Breadcrumbs/StyledBreadcrumbs.js

```diff
@@ -0,0 +1,37 @@
+import styled from 'styled-components'
+import {theme} from 'tocco-ui'
+
+import {StyledLink} from '../../../../components/StyledLink'
+
+export const StyledBreadcumbs = styled.div`
+  background-color: ${theme.color('backgroundBreadcrumbs')};
+  width: 100%;
+  padding: 1.3rem 1.7rem;
+  margin-bottom: 6px;
+  
+  span:nth-child(even) {
+    margin-left: 1.5rem;
+    margin-right: 1.5rem;
+  }
+  
+  span:last-child a {
```

Comment:
alternatively you could have set a property in the loop (Breadcrumbs.js).

` <StyledBreadcrumbsLink active={idx === breadcrumbsInfo.length -1} .../>`